### PR TITLE
(BIDS-2529) Add block column to tables on address page

### PR DESCRIPTION
**Pull request by D13ce**
This PR adds a block column to the tables on the address page, namely to
- Internal Txs
- Blob Txs (for Dencun)
- Erc20 Txs
- Erc721 Txs
- Erc1155 Txs

### CODE REVIEW

File: handlers/eth1Account.go
Comment (LuccaBitfly) [inline handlers/eth1Account.go:1]: Opening an address page on your branch, e.g. `/address/0x8be1a5737817d3c8bf65039a8d374f920819563a` on **Holesky**, results in some huge error for me, which does not happen on the master branch.

**Review by LuccaBitfly — CHANGES_REQUESTED** [review]
Everything seems to look fine, but

File: handlers/eth1Account.go
Comment (D13ce) [reply handlers/eth1Account.go:1]: What errors are you seeing? Does the page load?

**Review by LuccaBitfly — APPROVED** [review]
lgtm, pls don't hurt me

File: handlers/eth1Account.go
Comment (D13ce) [reply handlers/eth1Account.go:1]: As discussed, the error you were talking about was a warning and did not come from my PR.

**Pull request merged**